sdk/c/test/test.c

```diff
@@ -0,0 +1,26 @@
+/**************************************************************
+ * Copyright (c) 2018-2020,Hangzhou Rokid Tech. Co., Ltd.
+ * All rights reserved.
+ *
+ * FileName: test.c
+ * Description: Just test yoda c app.
+ *
+ * Date  :	2019.06.18
+ * Author:  
+ * Modification: add file
+ *
+ **************************************************************/
```

Comment:
YodaOS is a separate project from the company Rokid, so please remove the above comments.